src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,68 @@
+.stopwatch {
+  position: relative;
+  width: $width;
+  height: $width;
+  border: $border-width dotted $border-color;
+  border-radius: 50%;
+
+  &__minutes-hand {
+    position: absolute;
+    height: $minutes-hand-height;
+    width: $hand-width-minutes;
+    background-color: $minutes-hand-color;
+    bottom: 50%;
+    left: calc(50% - (#{$hand-width-minutes} / 2));
+    transform-origin: bottom center;
+    animation: spin-minute 3600s steps(60) infinite;
+  }
+
+  &__seconds-hand {
+    position: absolute;
+    height: $seconds-hand-height;
+    width: $hand-width-seconds;
+    background-color: $seconds-hand-color;
+    bottom: 50%;
+    left: calc(50% - (#{$hand-width-seconds} / 2));
```

Comment:
Similar to the comment for the minutes hand, ensure that the `$hand-width-seconds` variable is defined and has a unit that makes the `calc(50% - (#{$hand-width-seconds} / 2))` calculation valid in CSS.